ddp-workspace/projects/ddp-brain/src/assets/i18n/en.json

```diff
@@ -68,6 +68,12 @@
                 "Part2": "visit our site"
             }
         },
+        "EndEnrollment": {
+            "Title": "Thank you for your interest in the Brain Tumor Project",
+            "Timeline": {
+                "2018-text1": "Brain Tumor  launched through collaboration with patients and advocates.. from en.json"
```

Comment:
admittedly small nitpick but there's an extra space between 'Tumor' and 'launched'

Reply:
by chance, the text says 2018 (in 2018-text1) but the pic of the timeline in the Conversation tab shows 2019 (with similar text of project launching), typo maybe? (I honestly can't remember off the top of my head the exact year it was launched)